backend/.env.dev

```diff
@@ -1,6 +1,6 @@
 DEBUG=1
 SECRET_KEY=foo
-DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1] 0.0.0.0
+DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1] 0.0.0.0 212.182.24.47
```

Comment:
IP nie warto wstawiać na publiczne repo.
Teraz to już dupa ale lepiej to wstaw i tak do .env którego nie ma publicznie

Reply:
Później i tak możesz sprawdzić ip domeny, które będzie takie samo cnie?

Reply:
Ludzie korzystający z naszej apki tak, spoko. Ale scrapery już nie.